text/0000-sawtooth-governance.md

```diff
@@ -0,0 +1,348 @@
+  at the expense of another, and people on both sides feel strongly about it.
+
+In either case, an alternative form of decision-making is needed.
+
+* For the "trivial" case, usually either the RFC shepherd or subteam leader
+  will make an executive decision.
+
+* For the "deep" case, the subteam leader is empowered to make a final
+  decision, but should consult with the rest of the root team before doing so.
+
+#### Final Comment Period (FCP)
+
+Each RFC has a shepherd drawn from the relevant subteam. The shepherd is
+responsible for driving the consensus process -- working with both the RFC
+author and the broader community to dig out problems, alternatives, and
+improved design, always working to reach broader consensus.
+
+At some point, the RFC comments will reach a kind of "steady state", where no
+new tradeoffs are being discovered, and either objections have been addressed,
+or it's clear that the design has fundamental downsides that need to be
+weighed.
+
+At that point, the shepherd will announce that the RFC is in a "final comment
+period" (which lasts for one week). This is a kind of "last call" for strong
+objections to the RFC. **The announcement of the final comment period for an
+RFC should be very visible**; it should be included in the subteam's periodic
+communications.
+
+After the final comment period, the subteam can make a decision on the RFC. The
+role of the subteam at that point is *not* to reveal any new technical issues
+or arguments; if these come up during discussion, they should be added as
+comments to the RFC, and it should undergo another final comment period.
+
+Instead, the subteam decision is based on **weighing the already-revealed
+tradeoffs against the project's priorities and values** (which the root team is
+responsible for setting, globally). In the end, these decisions are about how
+to weight tradeoffs. The decision should be communicated in these terms,
+pointing out the tradeoffs that were raised and explaining how they were
+weighted, and **never introducing new arguments**.
+
+### Initial Subteams
+
+The following is a proposed list of initial subteams which is subject to change
+at the discretion of the root team, once it has formed.
+
+**Core** - Responsible for the design, architecture, performance, stability,
+ and security of the core Sawtooth platform
+
+**Application SDKs** - Responsible for the design and consistency of
+user-facing SDKs for application development
+
+**Consensus** - Responsible for the supported consensus algorithms and
+the consensus interface
+
+**Seth** - Responsible for the Sawtooth integration with Ethereum
+
+**Community Outreach** - Responsible for writing and hosting documentation,
+managing the main Sawtooth website, promoting Sawtooth to the public, managing
+demos, and providing the community with training and support
+
+**Release Management** - Responsible for release related issues such as
```

Comment:
I'm not convinced this should be a separate group? I think there are cross-cutting concerns like release and CI that should be handled by the top-level team, with sub-teams responsible for separate projects able to modify the processes within limits.

Reply:
For the subteam breakdown, we tried to take inspiration from an existing model that seems to be working: https://www.rust-lang.org/en-US/team.html Their breakdown includes high-level, cross-cutting concerns such as a release team and a community team.

Re-reading this description again, I think it could be clarified that this subteam is responsible for the release management of the core platform and not necessarily separate components like the SDKs and Seth.